_stbt/control.py

```diff
@@ -175,20 +204,35 @@ def __init__(self, control_name, connect_fn):
 
     def press(self, key):
         s = self._connect()
-        s.sendall("SEND_ONCE %s %s\n" % (self.control_name, key))
-        _read_lircd_reply(s)
+        command = "SEND_ONCE %s %s" % (self.control_name, key)
+        s.sendall(command + "\n")
+        _read_lircd_reply(s, command)
         debug("Pressed " + key)
 
+    def keydown(self, key):
+        s = self._connect()
+        command = "SEND_START %s %s" % (self.control_name, key)
+        s.sendall(command + "\n")
+        _read_lircd_reply(s, command)
+        debug("Holding " + key)
+
+    def keyup(self, key):
+        s = self._connect()
+        command = "SEND_STOP %s %s" % (self.control_name, key)
+        s.sendall(command + "\n")
+        _read_lircd_reply(s, command)
+        debug("Released " + key)
```

Comment:
Commit 5f040b7 from #73 said that we needed to keep the socket open until the keyup, but I don't see how that can be true considering that `irsend SEND_START ... ; irsend SEND_STOP ...` from the command-line works correctly, and it is using 2 entirely separate processes for the start & stop.